drivers/cmd/webhook/main_test.go

```diff
@@ -43,13 +44,14 @@ func TestExtractEvent(t *testing.T) {
 		Addr:    "127.0.0.1:8999",
 		Handler: http.HandlerFunc(hookHandlerTest),
 	}
-	daemon.Children.Add(1)
+	waitgroup.Add(1)
 	go func() {
-		defer daemon.Children.Done()
 		server.ListenAndServe()
+		waitgroup.Done()
```

Comment:
we should use the `defer` to execute this statement in case there is a crash in `server.ListenAndServe()`